transport/broker.go

```diff
@@ -18,7 +15,6 @@ func StartBroker(du string, ru string) error {
 	if err != nil {
 		return err
 	}
-	log.Debug("Dealer bound to " + du)
```

Comment:
Most of these logs...were probably useful for troubleshooting at this lower level...so I'm not too keen on deleting them.